examples/gagent-related-twice-to-same-app.py

```diff
@@ -0,0 +1,88 @@
+# ruff: noqa: D103
+# ruff: noqa: D100
+# ruff: noqa: F524
```

Comment:
[F524](https://docs.astral.sh/ruff/rules/string-dot-format-missing-arguments/) seems legit. What triggers it?